scripts/cipublish

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if [[ -n "${STAC4S_SERVER_DEBUG}" ]]; then
```

Comment:
My guess is this was borrowed from GeoTrellis Server and there was a <kbd>control + find and replace</kbd> operation that left this behind.